feat: adds status to the github-pull-requests-board (#22792)
backstage/backstage: .changeset/chilled-dolphins-tap.md

---
'@backstage/plugin-github-pull-requests-board': patch
---

> The cardheader component in the github-pull-requests-board plugin now requires that a `status` is passed to the component.

Reviewer: ```suggestion
The `CardHeader` component in the `github-pull-requests-board` plugin will show the status for the PR
```
